src/components/menus/content-contextual-menu.jsx

```diff
@@ -392,8 +392,8 @@ const ContentContextualMenu = (props) => {
     }, [selectedElements, userId]);
 
     const allowsDelete = useCallback(() => {
-        return isUserAllowed() && noCreationInProgress();
-    }, [isUserAllowed, noCreationInProgress]);
+        return isUserAllowed() && selectedElements.every((el) => el.elementUuid != null);
```

Comment:
With this change not only studies in creation will be allowed to be deleted but all types of elements, is that ok ?

Reply:
other types or elements don't have the same "oncreation" status so it doesn't affect them, we can always delete them

Reply:
Alright